tests/cpp/lp_namer/AntaresProblemToXpansionProblemTranslatorTest.cpp

```diff
@@ -1,83 +1,93 @@
-#include "gtest/gtest.h"
-
-#include "antares-xpansion/lpnamer/problem_modifier/AntaresProblemToXpansionProblemTranslator.h"
-
+#include <antares-xpansion/lpnamer/problem_modifier/RenameUtils.h>
 #include <string>
 #include <unordered_map>
 #include <vector>
 
-// forward declaration for testing internal renaming helper (defined in the .cpp)
-void rename_week_names(unsigned int week,
-                       const std::vector<std::string>& names,
-                       std::unordered_map<int, std::vector<std::string>>& container_names);
+#include "antares-xpansion/lpnamer/problem_modifier/AntaresProblemToXpansionProblemTranslator.h"
+#include "gtest/gtest.h"
 
-TEST(AntaresProblemToXpansionProblemTranslatorTest, convertSignToLEGSpan) {
-  std::vector<char> signs = {'<', '=', '>'};
-  std::vector<char> expected = {'L', 'E', 'G'};
-  std::vector<char> result = AntaresProblemToXpansionProblemTranslator::convertSignToLEG(std::span<char>(signs.data(), signs.size()));
-  ASSERT_EQ(result, expected);
+TEST(AntaresProblemToXpansionProblemTranslatorTest, convertSignToLEGSpan)
+{
+    std::vector<char> signs = {'<', '=', '>'};
+    std::vector<char> expected = {'L', 'E', 'G'};
+    std::vector<char> result = AntaresProblemToXpansionProblemTranslator::convertSignToLEG(
+      std::span<char>(signs.data(), signs.size()));
```

Comment:
`std::vector<T>` is (implicitly) convertible to `std::span<T>` so you may try
```suggestion
      AntaresProblemToXpansionProblemTranslator::convertSignToLEG(signs);
```
The implicit constructor uses `std::ranges::data` and `std::ranges::size` under the hood.